src/presentation/http/schema/User.ts

```diff
@@ -4,6 +4,7 @@
 export const UserSchema = {
   $id: 'UserSchema',
   type: 'object',
+  required: [ 'email' ],
```

Comment:
This change is unclear for me. For what case it marked as required? do we have any API routes that accepts a user? 

Or it prevents returning user without "email"? If so, I think it would be better to leave "email" optional, since it is a private data and probably not all api routes would return it